app/javascript/application.js

```diff
@@ -1,8 +1,16 @@
+import '@hotwired/turbo-rails';
+import './controllers';
 import React from 'react';
-import ReactDOM from 'react-dom';
+import ReactDOM from 'react-dom/client';
+import { Provider } from 'react-redux';
+import store from './react/redux/store';
+import App from './react/components/App';
 
-function App() {
-  return <h1>Hello World!</h1>;
-}
-
-ReactDOM.render(<App />, document.getElementById('root'));
+const root = ReactDOM.createRoot(document.getElementById('root'));
+root.render(
+  <React.StrictMode>
+    <Provider store={store}>
+      <App />
+    </Provider>
+  </React.StrictMode>,
+);
```

Comment:
- [ ] Kindy make sure a greeting message is displayed here. currently, I am just getting a black screen.

<img width="1040" alt="image" src="https://user-images.githubusercontent.com/65068771/229331046-aa9d9728-7d66-4769-a993-7705d3c02e90.png">

Reply:
It is displayin.
![rail-react](https://user-images.githubusercontent.com/60324033/229331251-bbff215a-4431-4a09-a37c-e82c75b657cf.PNG)
![rails-react-2](https://user-images.githubusercontent.com/60324033/229331261-ea665269-ffb5-4419-bb44-73e5fd7b9cf0.PNG)

Reply:
After following the instructions in your readme file the project is still not displaying.